src/components/atoms/skill-level-tiles.astro

```diff
@@ -0,0 +1,40 @@
+---
+export interface Props extends astroHTML.JSX.HTMLAttributes {
+  skillLevel: number;
+}
+const { skillLevel } = Astro.props;
+
+const Levels: number[] = [1, 2, 3, 4, 5];
+---
+
+<div class:list={['flex', 'flex-row', 'justify-between']}>
+  {
+    Levels.map((level) =>
+      level > skillLevel ? (
+        <div
+          class:list={[
```

Comment:
Jedyna klasa, która się zmienia to ta odpowiedzialna za kolor. Możesz w ramach `class:list` wykorzystać ternary operator, żeby podmienić tylko ją zamiast renderować dwa osobne div'y